src/components/badge/Badge.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+
+import { Body } from '../typography/Body';
+import { ThemType, useTheme } from '../../styles/themes';
+import type { WithTestID } from 'src/shared/type';
+
+type BadgeProps = WithTestID<{
+    number: number;
+    maxSize: number;
+    backgroundColor?: string;
+    color?: string;
+}>;
+
+const DEFAULT_COLOR = '#ffffff';
+const DEFAULT_BACKGROUND_COLOR = '#2F4158';
+
+const createStyle = (
+    theme: ThemType,
+    color: string,
+    backgroundColor: string,
+) => {
+    return StyleSheet.create({
+        badge: {
+            backgroundColor,
+            color,
+            borderRadius: 8,
+            paddingHorizontal: theme.sw.spacing.xs,
+            paddingVertical: theme.sw.spacing.xxs,
+            fontSize: 14,
+        },
+    });
+};
+
+export const Badge = ({
+    number,
+    maxSize,
+    testID,
+    backgroundColor = DEFAULT_BACKGROUND_COLOR,
+    color = DEFAULT_COLOR,
+}: BadgeProps) => {
+    const theme = useTheme();
+    const displayText = getTruncatedNumber(number, maxSize);
+    const badgeStyle = createStyle(theme, color, backgroundColor).badge;
+
+    return (
+        <View style={{ alignSelf: 'flex-start' }} testID={testID}>
+            <Body size='B2' weight='semi-bold' style={badgeStyle}>
+                {displayText}
+            </Body>
+        </View>
+    );
+};
+
+function getTruncatedNumber(number: number, maxSize: number): string {
```

Comment:
Je ne pense pas qu'il faille mettre cette logique dans le Design System.
C'est du métier spécifique à son implémentation.

Reply:
Je suis pas forcément d'accord, pour moi ça peut être une fonctionnalité du composant lui-même

Reply:
Dans ce cas, c'est à @lauren-sw de trancher.

Reply:
Alors pour le coup je rejoins en partie Clément sur un truc: cette max size ne devrait pas être un paramètre obligatoire, et s'il n'est pas renseigné on ne tronque pas. (un peu comme les max number of lines de composant natifs genre label ou autre)